exercises/hamming/hamming_test.exs

```diff
@@ -8,34 +8,39 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule DNATest do
   use ExUnit.Case
 
-  # @tag :pending
+#   @tag :pending
+  test "hamming distance for strands of the same length are ok" do
+    assert {:ok, _} = DNA.hamming_distance('', '')
+  end
+
+  @tag :pending
+  test "hamming distance is undefined for strands of different lengths" do
+    assert {:error, "Lists must be the same length."} = DNA.hamming_distance('AAAC', 'TAGGGGAGGCTAGCGGTAGGAC')
+    assert {:error, "Lists must be the same length."} = DNA.hamming_distance('GACTACGGACAGGACACC', 'GACATCGC')
+  end
+
+  @tag :pending
   test "no difference between empty strands" do
-    assert DNA.hamming_distance('', '') == 0
+    assert {:ok, 0} = DNA.hamming_distance('', '')
```

Comment:
With the exception of that first test the rest of these can be `DNA.hamming_distance('', '') == {:ok, 0}` which is consistent with the other exercises.